### PR TITLE
feat: add "align" and "sort" options

### PR DESCRIPTION
**Pull request by rquadling**
The purpose of this change is to provide a level of predictability in the location of the entries in an INI file when they are written to a file. In addition, the alignment of the values based upon the `=` character provides a small visibility improvement.

## References

Implements #192

### CODE REVIEW

File: lib/ini.js [hunk 1]
```diff
@@ -4,27 +4,46 @@ const encode = (obj, opt = {}) => {
...
+  const keys = sort ? Object.keys(obj).sort() : Object.keys(obj)
+
+  // padToChars is determined as follows:
+  // 1. Get the keys
+  // 2. Exclude keys pointing to objects unless the value is null or an array
+  // 3. Add `[]` to array keys
+  // 4. Ensure non empty set of keys
+  // 5. Reduce the set to the longest key
+  // 6. Get the length
+  const padToChars = opt.align
```
Comment (wraithgar) [inline lib/ini.js:29]: Nested ternaries are very hard to follow and prone to errors.  Can this be a `let` that defaults to `0` and an if statement that changes it if `opt.align` is set?

Comment (rquadling) [reply lib/ini.js:29]: Yep. I was being a little lazy here I guess. JS is not my first or second language so always happy to accept coding style recommendations.

File: test/foo.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ const fs = require('fs')
 const path = require('path')
 const fixture = path.resolve(__dirname, './fixtures/foo.ini')
 const data = fs.readFileSync(fixture, 'utf8')
+const jsonFixture = path.resolve(__dirname, './fixtures/bar.json')
```
[the remark this answers is not in the document]
Comment (rquadling) [reply test/foo.js:8]: But I've swapped over to the `foo.ini` set and found an issue with my code (I'd not used `safe()` on the keys so missed `"<key>"` quoting). Will now see if enough of the sort/align checking is visible in the tests.

[the remark this answers is not in the document]
Comment (wraithgar) [reply test/foo.js:8]: Add missing cases to foo.ini if there are any.

**Review by wraithgar — APPROVED** [review]
Thank you. Will land once CI tests finish.
